src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -28,6 +39,80 @@ export const ConfirmBidRoute: React.FC<BidProps> = props => {
     saleArtwork,
   })
 
+  function createBidderPosition(maxBidAmountCents: number) {
+    return new Promise(async (resolve, reject) => {
+      commitMutation<ConfirmBidCreateBidderPositionMutation>(
+        props.relay.environment,
+        {
+          onCompleted: data => {
+            resolve(data)
+          },
+          onError: error => {
+            reject(error)
+          },
+          mutation: graphql`
+            mutation ConfirmBidCreateBidderPositionMutation(
+              $input: BidderPositionInput!
+            ) {
+              createBidderPosition(input: $input) {
+                result {
+                  status
+                  message_header
+                  message_description_md
+                }
+              }
+            }
+          `,
+          variables: {
+            input: {
+              sale_id: sale.id,
+              artwork_id: artwork.id,
+              max_bid_amount_cents: maxBidAmountCents,
+            },
+          },
+        }
+      )
+    })
+  }
+
+  function handleMutationError(actions: FormikActions<object>, error: Error) {
+    logger.error(error)
+
+    let errorMessages: string[]
+    if (Array.isArray(error)) {
+      errorMessages = error.map(e => e.message)
+    } else if (typeof error === "string") {
+      errorMessages = [error]
+    } else if (error.message) {
+      errorMessages = [error.message]
+    }
+
+    // TODO: add tracking with errorMessages
+    logger.error(errorMessages) // remove once tracking is implemented
```

Comment:
We are calling `logger.error` twice and I would remove the second one since `errorMessages` is always a string whereas `error` could be an error object that has more information such as backtrace.

Reply:
The 2nd one will get removed and replaced with a call to tracking as soon as this is merged and I can work on [that ticket](https://artsyproduct.atlassian.net/browse/AUCT-667). I only added it to prevent a compile error from an unused variable.